### ISSUE TITLE
[Bug]: No context menu on tray icon (but click event works!)

### ISSUE THREAD

**Issue by axkibe** [electron/electron#32879]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

17.0.0

### What operating system are you using?

Other Linux

### Operating System Version

debian bullseye

### What arch are you using?

x64

### Expected Behavior

I created a simple tray icon. There is no context menu when clicked. Albeit the click event actually works (despite it should be broken you seem to have that fixed?).

I tried to force the contextmenu to pop up in the click event, but no avail. 

minimal testcase:
```
const { app, Menu, Tray } = require( 'electron' );
const path = require( 'path' );

app.on( 'ready', ( ) => {
    const tray = new Tray( path.join( __dirname, 'icon.png' ) );
    const contextMenu = Menu.buildFromTemplate([
        { label: 'Show', click: ( ) => console.log( 'show clicked' ) },
    ]);
    tray.setContextMenu( contextMenu );
    tray.on( 'click', ( ) => {
        console.log( 'click' );
        tray.popUpContextMenu( );
    } );
} );
```

PS: I'm using Gnome desktop, and libappindicator is installed (it makes no difference, the tray itself works without just as fine). To get top icons, I'm used "TopIcons Plus".

PS: RocketChat Desktop -- an electron application -- installed from flatpak the context menu works, but when I clone their most recent source code and start it with 'npm start', same behavior, no context menu.

### Actual Behavior

In short words, I expect a context menu when clicking the tray icon (tried left or right). I do get a click event, but context menu.

**Comment by axkibe** [comment]
Hmm, after logging out and in it works* now, my GNOME session must have come in some awkward state because of reasons.

* (Left click creates a click event, right click creates a context menu, fair enough for me)

Sorry for noise.
